app/routes.js

```diff
@@ -46,43 +43,50 @@ module.exports = function(app) {
             .exec(resHandler(res));
     });
 
+    // get specific field from screen document
+    app.get('/api/screen/:id/:field', function(req, res) {
+        Screen
+            .find({'_id': req.params.id})
+            .select(req.params.field)
+            .exec(resHandler(res));
+    });
+
     // get all samples belonging to screen
     app.get('/api/samples/:screen', function(req, res) {
         Sample
         .find({ 'screen': req.params.screen })
-        .select('_id row column plate gene_name feature_vector_std ' +
+        .select('_id row column plate gene_name ' +
                 'pca_vector neighbours')
         .exec(resHandler(res));
     });
 
-    // get specific sample
-    app.get('/api/sample/:id', function(req, res) {
-          Sample
-            .find({ '_id': req.params.id })
-            .exec(resHandler(res))
+    // get value from sample document
+    app.get('/api/sample/:id/:field', function(req, res) {
+      Sample
+        .find({ '_id': req.params.id })
+        .select(req.params.field)
+        .exec(resHandler(res))
```

Comment:
@starcalibre are you saying you're punting on not displeasing me with the huge number of anons? =)

Incidentally, #52 becomes much, much easier if you create functions (only need to modify the function) vs using anons (need to modify each and every anon).